From a117af244f8c1e568f4921fad2d6c1ae06b74a86 Mon Sep 17 00:00:00 2001
From: Andreas Metzler <ametzler@debian.org>
Date: Thu, 9 May 2024 13:57:27 +0200
Subject: Do not set AEAD: OCB key preference on new keys.

Origin: vendor
Forwarded: not-needed
Last-Update: 2024-05-09
---
 g10/keygen.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/g10/keygen.c b/g10/keygen.c
index 26126cde7..773709ec5 100644
--- a/g10/keygen.c
+++ b/g10/keygen.c
@@ -523,6 +523,10 @@ keygen_set_std_prefs (const char *string,int personal)
 	xfree (prefstringbuf);
       }
 
+    /* Do not set AEAD: OCB on newly generated key, it is a LibrePGP
+     * feature.  */
+    ocb = 0;
+
     if(!rc)
       {
 	if(personal)
-- 
2.43.0

